### DISCUSSION TITLE
Framework other

### DISCUSSION THREAD

**Discussion by advany** [vercel/vercel#8179] [Help] [answered]
I create deployments using the vercel api. That worked perfectly until couple hours ago it forced me to fill in the framework (https://vercel.com/docs/rest-api#endpoints/projects/create-a-new-project). I am not able to fill in null or other. I don't use a framework...

**Comment by amyegan** [comment, marked as answer]
Hi @advany. I just tried creating a no-framework project using the API. I get an "invalid_framework" error if I use quotes around the framework value (like `"null"`), but it does work for me if I leave them out to provide a true null value (like `null`)

<img width="733" alt="Screen Shot 2022-07-18 at 11 01 02 AM" src="https://user-images.githubusercontent.com/8247105/179542610-073726f1-3460-459a-adfa-a9fe44957d23.png">

If it still isn't working for you after changing the framework value to `null`, please let me know and I will look into other possible causes.

**Reply by advany** [reply]
Thnx @amyegan! Maybe the error can be improved that a true null is also an option.

**Reply by amyegan** [reply]
I have an internal ticket for to investigate how we can improve clarity on this. I'm glad we got it working for you!
